app/api/models/karyotype.py

```diff
@@ -24,4 +23,4 @@ class Region(BaseModel):
 
 
 class Karyotype(BaseModel):
-    top_level_regions: List[Region]
+    top_level_regions: list[Region]
```

Comment:
I guess this was Pedantic List and not python list.

Reply:
The `List` and `Dict` types here and elswhere were imported from python `typing` module (not needed in python 3.8+).